unistd/file.c

```diff
@@ -268,10 +257,15 @@ int create_dev(oid_t *oid, const char *path)
 					tpath = tpathalloc;
 				}
 
+				/* TODO: not needed, we may assume it will always be canonical? */
+#if 0
```

Comment:
@agkaminski : I don't know how `create_dev()` is being used (it would be nice to add some unit tests with standard use cases.

Please comment.

Reply:
leaving as-is, to be done in separate PR